File: BigONotation/O(n^2)/main.py
def log(num):
    for number in num:  # O(n)
        print(number)
    # O(n^2)
    for first in num:  # O(n)
        for second in num:  # O(n)
            print(first + ', ' + second)


def go(num):
    # O(n^3)
    for first in num:  # O(n)
        for second in num:  # O(n)
            for third in num:  # O(n)
                print(first + ', ' + second + ',' + third)

File: BigONotation/O(n^2)/main.js
let log = function (num) { // O(n^2)
    for (let number of num) console.log(number); //O(n)
    // O(n^2)
    for (let first of num) { // O(n)
        for (let second of num) { // O(n)
            console.log(first + ',' + second)
        }
    }
}

let go = function (num) {
    // O(n^3)
    for (let first of num) { // O(n)
        for (let second of num) { // O(n)
            for (let third of num) { // O(n)
                console.log(first + ',' + second + ',' + third)
            }
        }
    }
}